app/javascript/mastodon/locales/fr.json

```diff
@@ -87,7 +87,7 @@
   "confirmations.mute.confirm": "Masquer",
   "confirmations.mute.message": "Confirmez-vous le masquage de {name} ?",
   "confirmations.redraft.confirm": "Effacer et ré-écrire",
-  "confirmations.redraft.message": "Êtes vous sûr de vouloir effacer ce statut pour le ré-écrire ? Vous perdrez toutes ses réponses, ses repartages et ses mises en favori.",
+  "confirmations.redraft.message": "Êtes vous sûr de vouloir effacer ce statut pour le ré-écrire ? Vous perdrez toutes ses réponses, ses repartages et ses mises en favori.",
```

Comment:
related: #8246 
A new text has been suggested and approved by gargron. How to process, should we write the new french translation here?

Reply:
Thank you for telling me that!
What I suggest: once this PR get merged, I will make another one that update both the English version and the French one. Does that sounds good to you?

Reply:
"êtes-vous"; non ? (avec le trait d'union)

Reply:
I would even suggest «Êtes-vous sur·e» ;)

Reply:
Sounds good for me @Sylvhem